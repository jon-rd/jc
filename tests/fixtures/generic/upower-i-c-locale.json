[{"native_path": "/sys/devices/LNXSYSTM:00/device:00/PNP0C0A:00/power_supply/BAT0", "vendor": "NOTEBOOK", "model": "BAT", "serial": "0001", "power_supply": true, "updated": "Thu Feb 9 18:42:15 2012", "has_history": true, "has_statistics": true, "detail": {"type": "battery", "present": true, "rechargeable": true, "state": "charging", "energy": 22.3998, "energy_empty": 0.0, "energy_full": 52.6473, "energy_full_design": 62.16, "energy_rate": 31.6905, "voltage": 12.191, "time_to_full": 57.3, "percentage": 42.5469, "capacity": 84.6964, "technology": "lithium-ion", "energy_unit": "Wh", "energy_empty_unit": "Wh", "energy_full_unit": "Wh", "energy_full_design_unit": "Wh", "energy_rate_unit": "W", "voltage_unit": "V", "time_to_full_unit": "minutes"}, "history_charge": [{"time": 1328809335, "percent_charged": 42.547, "status": "charging"}, {"time": 1328809305, "percent_charged": 42.02, "status": "charging"}, {"time": 1328809275, "percent_charged": 41.472, "status": "charging"}, {"time": 1328809245, "percent_charged": 41.008, "status": "charging"}], "history_rate": [{"time": 1328809335, "percent_charged": 31.691, "status": "charging"}, {"time": 1328809305, "percent_charged": 32.323, "status": "charging"}, {"time": 1328809275, "percent_charged": 33.133, "status": "charging"}], "updated_seconds_ago": 1, "updated_epoch": 1328841735, "updated_epoch_utc": null}]
