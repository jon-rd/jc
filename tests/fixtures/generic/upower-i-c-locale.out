  native-path:          /sys/devices/LNXSYSTM:00/device:00/PNP0C0A:00/power_supply/BAT0
  vendor:               NOTEBOOK
  model:                BAT
  serial:               0001
  power supply:         yes
  updated:              Thu Feb  9 18:42:15 2012 (1 seconds ago)
  has history:          yes
  has statistics:       yes
  battery
    present:             yes
    rechargeable:        yes
    state:               charging
    energy:              22.3998 Wh
    energy-empty:        0 Wh
    energy-full:         52.6473 Wh
    energy-full-design:  62.16 Wh
    energy-rate:         31.6905 W
    voltage:             12.191 V
    time to full:        57.3 minutes
    percentage:          42.5469%
    capacity:            84.6964%
    technology:          lithium-ion
  History (charge):
    1328809335  42.547  charging
    1328809305  42.020  charging
    1328809275  41.472  charging
    1328809245  41.008  charging
  History (rate):
    1328809335  31.691  charging
    1328809305  32.323  charging
    1328809275  33.133  charging

